reporter/src/funTest/assets/file-counter-expected-scan-report.html

```diff
@@ -301,7 +295,7 @@ <h3>Packages</h3>
     <td><ul>
         <li><p>n/a: FileCounter - DownloadException: No source artifact URL provided for 'Gradle:com.here.ort.gradle.example:lib:1.0.0'.<br/>Caused by: DownloadException: No VCS URL provided for 'Gradle:com.here.ort.gradle.example:lib:1.0.0'. Please make sure the release POM file includes the SCM connection, see: https://docs.gradle.org/current/userguide/publishing_maven.html#example_customizing_the_pom_file</p><p></p></li>
     </ul></td>
-</tr><tr class="ort-success">
+</tr><tr class="">
```

Comment:
reason was consistency with the other code

Reply:
maybe more importantly the code seems simpler

Reply:
How is `<tr class="">` simpler than `<tr>`? I think it's better to remove garbage before it makes its way into master, otherwise it might stay there forever, especially because removing these should not take much time, probably less than writing these comments.

Reply:
with simpler i was referring to the code generating the html, not the generated html.
Anyhow, done.

Reply:
Ah, I didn't think of that.

Reply:
re-added the success class in the end - and thomas will soon style it a bit using this class.